packages/core/src/plugins/PastePlugin.ts

```diff
@@ -0,0 +1,13 @@
+import { Plugin, PluginKey } from '@tiptap/pm/state'
+
+export const PastePlugin = (onPaste: (e: ClipboardEvent) => void) => {
+  return new Plugin({
+    key: new PluginKey('tiptapPaste'),
+
+    props: {
+      handlePaste: (_, e) => {
+        onPaste(e)
+      },
```

Comment:
On looking at this again, would we want to also pass through any of these other values?

I see that `handlePaste` is defined as:
```
    /**
    Can be used to override the behavior of pasting. `slice` is the
    pasted content parsed by the editor, but you can directly access
    the event to get at the raw content.
    */
    handlePaste?: (this: P, view: EditorView, event: ClipboardEvent, slice: Slice) => boolean | void;
```
I think at least `slice` might be useful to passthrough?

Reply:
Good idea - done.
Added `slice` for onPaste and `slice` and `moved` for onDrop.